src/App.js

```diff
@@ -1,24 +1,74 @@
-import logo from './logo.svg';
-import './App.css';
+import Navbar from "./Components/Navbar";
+import { useEffect, useState } from "react";
+import { fetchFolderData, fetchUserData } from "./api";
+import Profile from "./Components/Profile";
+import CardList from "./Components/CardList";
+import Footer from "./Components/Footer";
+import { Route, Routes } from "react-router-dom";
+import Detail from "./Components/Detail";
 
 function App() {
+  const [user, setUser] = useState({
+    id: null,
+    name: null,
+    email: null,
+    profileImageSource: null,
+  });
```

Comment:
@HMRyu 

각 필드마다 null로 두기 보단, user라는 상태를 nullable로 관리하는게 좋아보여요
```suggestion
  const [user, setUser] = useState(null)
```